pkg/docker/containers-view.jsx

```diff
@@ -129,6 +129,29 @@ var ContainerDetails = React.createClass({
     }
 });
 
+var ContainerProblems = React.createClass({
+    onItemClick: function (event) {
+        cockpit.jump(event.currentTarget.dataset.url, cockpit.transport.host);
+    },
+
+    render: function () {
+        var problem = this.props.problem;
+        var problem_cursors = [];
+        for (var i = 0; i < problem.length; i++) {
+            problem_cursors.push(<a data-url={problem[i][0]} className='list-group-item' onClick={this.onItemClick}>
+            <span className="pficon pficon-warning-triangle-o fa-lg"></span>
+            {problem[i][1]}
```

Comment:
These two lines should be indented for easier reading, and representing the nesting inside the <a>.